### PR TITLE
BUG: Set all component of 3D points

### PR DESCRIPTION
**Pull request by blowekamp**
This addresses valgrind defects related to using uninitialized values
related to the 3d component of the point being unset.

### CODE REVIEW

**Comment by blowekamp** [comment]
After this is merged I expect Valgrind to be clean.

**Comment by dzenanz** [comment]
> I expect Valgrind to be clean

Awesome!

File: Examples/Filtering/SpatialObjectToImage1.cxx [hunk 1]
```diff
@@ -204,12 +206,14 @@ int main( int argc, char *argv[] )
...
   point[0] =  size[0] * 0.2 * spacing[0];
   point[1] =  size[1] * 0.8 * spacing[1];
+  point[2] =  size[2] * 0.2 * spacing[2];
   tubePoint.SetPositionInObjectSpace( point );
   tubePoint.SetRadiusInObjectSpace( size[0] * 0.05 * spacing[0] );
   tubePointList.push_back( tubePoint );
 
   point[0] =  size[0] * 0.8 * spacing[0];
   point[1] =  size[1] * 0.8 * spacing[1];
+  point[2] =  size[2] * 0.8 * spacing[1];
```
Comment (N-Dekker) [inline Examples/Filtering/SpatialObjectToImage1.cxx]: @blowekamp Just checking... shouldn't that be `spacing[2]`? (Not sure if it matters, it's just an example, right?)

Comment (dzenanz) [reply Examples/Filtering/SpatialObjectToImage1.cxx]: All of us missed it 😄

**Pull request merged**